app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListFragment.kt

```diff
@@ -237,24 +237,28 @@ class ThreadListFragment : Fragment(), SwipeRefreshLayout.OnRefreshListener {
         }
     }
 
-    private fun performSwipeActionOnThread(swipeAction: SwipeAction, thread: Thread): Boolean {
-        return when (swipeAction) {
+    private fun performSwipeActionOnThread(swipeAction: SwipeAction, thread: Thread): Boolean = with(mainViewModel) {
```

Comment:
Add a comment or rename this function so that we understand that we delete the element from the list when it is false